### DISCUSSION TITLE
I was approved the student benefits but NO E-mail sent to me.

### DISCUSSION THREAD

**Discussion by javierdev776** [community/community#55892] [GitHub Education]
Hi everyone,
I was approved the student benefits but NO E-mail sent to me.
Approved Infromation: Congrats! Your request for @javierdev776 was approved and your benefits will be available soon.
Another Infromation: Hi, ***! You were last verified as a student on yesterday. It is not necessary for you to reverify at this time. There may be a wait period between verification and access to academic benefits.

I wait all day without any reply.
I would like to know how long I need to wait after "approved". Will I receive an email from Github? Will it be sent to my main email address, my backup email address or still my application email address？
Repeat: I ask a question if it has been Approved rather than rejected。

**Comment by tuves** [comment]
Welcome to the GitHub Community, @javierdev776, we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!
